DisplayCAL/display_cal.py

```diff
@@ -9502,16 +9502,9 @@ def measurement_report_consumer(
         # set Lab values
         labels_Lab = ("LAB_L", "LAB_A", "LAB_B")
         for data in (ti3_ref, ti3_joined):
-            if (
-                "XYZ_X" in list(data.DATA_FORMAT.values())
-                and "XYZ_Y" in list(data.DATA_FORMAT.values())
-                and "XYZ_Z" in list(data.DATA_FORMAT.values())
-            ):
-                if (
-                    "LAB_L" not in list(data.DATA_FORMAT.values())
-                    and "LAB_A" not in list(data.DATA_FORMAT.values())
-                    and "LAB_B" not in list(data.DATA_FORMAT.values())
-                ):
+            data_formats = list(data.DATA_FORMAT.values())
+            if b"XYZ_X" in data_formats and b"XYZ_Y" in data_formats and b"XYZ_Z" in data_formats:
```

Comment:
This is good but please keep the format, or better run "black" on this file

Reply:
black doesn't change the above, it does change assorted other parts of this file though so I can PR that if you like